modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/sync/PodcastSyncProcess.kt

```diff
@@ -1053,10 +1057,22 @@ class PodcastSyncProcess(
 
                 // don't update if times are very close
                 val currentUpTo = episode.playedUpTo
-                if (playedUpTo < currentUpTo - 2 || playedUpTo > currentUpTo + 2) {
+                if (playedUpTo < currentUpTo - 5 || playedUpTo > currentUpTo + 2) {
                     episode.playedUpTo = playedUpTo
                     episode.playedUpToModified = null
                     if (episodeInPlayer) {
+                        val diffSeconds = (playedUpTo - currentUpTo).roundToInt()
```

Comment:
It might be good to include a comment here as it's slightly confusing. How about something like: 
```
// when the episode is paused in the player, track if the position is skipping back. This is to help debug the playback jumping issue. 
```

This is going to track all skip backwards even if it's the user. Should we limit it by duration? Or maybe it doesn't matter. 🤔

Reply:
Good idea! Added a comment here: [fb564d8](https://github.com/Automattic/pocket-casts-android/pull/2292/commits/fb564d83d3e91b075b94ff9b893d8aee76e63ca6)